### PR TITLE
[percentile] Add validation method and fuzz testing for gkarray

### PR DESCRIPTION
**Pull request by jeerim**
### What does this PR do?

Add a validation method and fuzz testing to GKArray to ensure that invalid sketches do not cause panics.

### Motivation

Sketches with invalid state should not cause a panic when being aggregated/queried.

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: pkg/legacy/importer_test.go [hunk 1]
```diff
@@ -1,50 +0,0 @@
...
-// This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2017 Datadog, Inc.
-
-package legacy
-
-import (
-	"fmt"
-	"testing"
-
-	"github.com/DataDog/datadog-agent/pkg/collector/py"
```
Comment (GeorgeErickson) [inline pkg/legacy/importer_test.go]: This file doesn't seem to be related, bad merge?

Comment (jeerim) [reply pkg/legacy/importer_test.go]: Yes, you're right - fixing.

**Review by truthbk — APPROVED** [review]
Looks good to me! :shipit:
